tests/ui/js/base-page-storyEditTests.js

```diff
@@ -131,16 +138,21 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             }
         ],
         "tags": ["cute", "cats"],
+        "timestampCreated": "",
```

Comment:
Don't forget to update `timestampCreated`, `timestampLastModified` and `timestampPublished` to proper values at proper stages.

Reply:
Good catch, I'd forgotten about these three :) Thanks!